Server/helpers/Validation.js

```diff
@@ -29,4 +29,27 @@ const validateLoginInfo = (userData) => {
   return schema.validate(userData);
 };
 
-module.exports = { validateEmail, validateLoginInfo };
+const validatesignupInfo = (userData) => {
+  const schema = joi.object({
+    email: joi
+      .string()
+      .min(6)
+      .required()
+      .email(),
+    userName: joi
+      .string()
+      .min(5)
+      .max(30)
```

Comment:
change the character range for username, if someone his name contain 3 characters cannot register in this application !!!!